changelogs/fragments/6385.yml

```diff
@@ -0,0 +1,2 @@
+feat:
+- [MDS] Support for Timeline ([#6385](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/6385))
```

Comment:
While this is nice and short, it is not clear enough what impact users should expect.

Reply:
updated